recipe/meta.yaml

```diff
@@ -74,6 +75,8 @@ outputs:
     build:
       noarch: python
     requirements:
+      build:
+        - {{ stdlib("c") }}
```

Comment:
FWIW, this is a corner case that the bot does not cover very well, and could/should be removed.